core/src/com/po/fuck/Assets.java

```diff
@@ -6,34 +6,36 @@
 import com.badlogic.gdx.utils.JsonReader;
 import com.badlogic.gdx.utils.JsonValue;
 
-import java.util.HashMap;
-import java.util.Map;
+/**
+* assetsData.json description
+* each .png stores some table of frames for the object
+* path - path to texture
+* frameWidth - the width of the single frame
+* frameHeight - the height of the single frame
+* width - the total width of this texture
+* height - the total height of this texture
```

Comment:
So what is the difference between `frameWidth` and `width` ?

Reply:
.png files stores all frames in table form, so ```frameWidth``` it is the width of 1 frame, and ```width``` it is the width of the .png file.

Reply:
Can't we just obtain from png ?

Reply:
We can